packages/specification/schemas/components/MedicationRequest/DispenseNotificationMedicationRequest.yaml

```diff
@@ -31,84 +31,54 @@ properties:
         - $ref: ../_extensions/ControlledDrug.yaml
         - $ref: ../_extensions/DMResponsiblePractitioner.yaml
         - $ref: ../_extensions/DispensingInformation.yaml
-        - $ref: ../_extensions/MedicationRepeatInformation.yaml
+        - $ref: ../_extensions/DispensingMedicationRepeatInformation.yaml
         - $ref: ../_extensions/NHSBSAPrescriptionType.yaml
         - $ref: ../_extensions/PrescriptionEndorsement.yaml
   identifier:
     type: array
-    description: The unique identifier of the line item that is being dispensed.
+    description: The unique identifier for this line item on the prescription.
     items:
-      type: object
-      properties:
-        system:
-          type: string
-          description: Codesystem URL for the line item ID.
-          enum: [https://fhir.nhs.uk/Id/prescription-order-item-number]
-        value:
-          type: string
-          description: The unique ID for a given line item.
-          example: dd11b07e-e137-488e-bc28-ab50384208b5
+      $ref: ../_fragments/identifiers.yaml#/components/schemas/PrescriptionOrderItemNumber
   status:
     type: string
-    description: The status of a MedicationRequest object.
+    description: |
+      The status of a MedicationRequest object. 
+      This will be consistent across all MedicationRequests in a Bundle.
```

Comment:
This language suggests that we will be making sure it's consistent, but we need them to do that. Probably better to say "This should be consistent..."